### PR TITLE
Fixed delete integration test

### PR DESCRIPTION
**Pull request by RokasSakys**
This branch was created to ignore the chaos of IntegrationTesting by bringing only the intended changes.

### CODE REVIEW

File: Cheaper Effort/internalLog.txt [hunk 1]
```diff
@@ -955,3 +955,28 @@
 2022-12-06 21:08:37.3600 Warn Error has been raised. Exception: NLog.NLogConfigurationException: Target 'eventlog' not found for logging rule: *.
 2022-12-06 21:08:37.4457 Info Validating config: TargetNames=file, ConfigItems=24, FilePath=/Users/dianapopkovaite/Downloads/Cheaper-Effort/Cheaper Effort/bin/Debug/net6.0/NLog.config
 2022-12-06 21:08:38.4645 Info Validating config: TargetNames=file, ConfigItems=24, FilePath=/Users/dianapopkovaite/Downloads/Cheaper-Effort/Cheaper Effort/bin/Debug/net6.0/NLog.config
+2022-12-19 23:33:26.7905 Info Message Template Auto Format enabled
```
Comment (MazIeva) [inline Cheaper Effort/internalLog.txt:958]: don't upload InternalLog next time!

File: tests/IntegrationTests/RecipeDetailsTest.cs [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            .CreateClient(new WebApplicationFactoryClientOptions
+            {
+                AllowAutoRedirect = false
+            });
+        }
+
+        [Fact]
+        public async Task Delete_RemovesRecipeById_ReturnsOK()
+        {
+            // Arrange
```
Comment (MazIeva) [inline tests/IntegrationTests/RecipeDetailsTest.cs:65]: Good clear commenting!

**Review by MazIeva — COMMENTED** [review]
Looks good!

**Comment by popkovaite** [comment]
everything seems good, nice work:)

**Pull request merged**